### PR TITLE
#159987686 Implement API documentation with Swagger

### PR DESCRIPTION
**Pull request by lauragift21**
#### What does this PR do?
Setup Swagger for documenting API 
#### Description of Task to be completed?
- Install `swagger-ui-express`
- Generate a `swagger.json` file where all API logic for swagger will be documented.
#### How should this be manually tested?
- run `npm install`
- run `npm start`
- navigate to `localhost:3000/api-docs`
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#159987686](https://www.pivotaltracker.com/story/show/159987686)
#### Screenshots (if appropriate)
image

### CODE REVIEW

File: server/docs/swagger.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                    "type": "string"
+                },
+                "password": {
+                    "type": "string"
+                }
+            }
+
+        }
+    }
+}
```
Comment (emekafredy) [inline server/docs/swagger.json]: Across all the users' endpoints, you need to add more attributes for the Users profile. Check the Pivotal Tracker storyboard for this story [159987699](https://www.pivotaltracker.com/story/show/159987699)

**Review by emekafredy — COMMENTED** [review]
Across all users' endpoints, you need to add some users' attributes in addition to the existing ones. Check the Pivotal Tracker storyboard for this story [159987699](https://www.pivotaltracker.com/story/show/159987699)

File: server/docs/swagger.json (hunk 1 above)
Comment (lauragift21) [reply server/docs/swagger.json]: Thank you for the feedback @emekafredy. I'll check the storyboard and update the changes where applicable.

Comment (d-beloved) [reply server/docs/swagger.json]: You can go ahead to extend the coverage of the documentation by adding the docs for articles. Look through the Pivotal tracker storyboard for this project to get an idea for what is needed to be added.

**Review by d-beloved — COMMENTED** [review]
You can go ahead to extend the coverage of the documentation by adding the docs for articles. Look through the Pivotal tracker storyboard for this project to get an idea for what is needed to be added.

File: server/docs/swagger.json (hunk 1 above)
Comment (lauragift21) [reply server/docs/swagger.json]: Alright, duly noted @d-beloved.

**Pull request merged**